vulnfeeds/utility/logginghelpers.go

```diff
@@ -13,23 +13,21 @@ import (
 // CreateLoggerWrapper creates and initializes the LoggerWrapper,
 // and also returns a cleanup function to be deferred
 func CreateLoggerWrapper(name string) (LoggerWrapper, func()) {
-	projectId, ok := os.LookupEnv("GOOGLE_CLOUD_PROJECT")
-	var loggerOptions []logging.LoggerOption
-	if !ok {
-		log.Println("GOOGLE_CLOUD_PROJECT not set, routing logs to stdout")
-		loggerOptions = append(loggerOptions, logging.RedirectAsJSON(os.Stdout))
+	projectId, projectIdSet := os.LookupEnv("GOOGLE_CLOUD_PROJECT")
+	if !projectIdSet {
+		return LoggerWrapper{}, func() {}
```

Comment:
I'm having a hard time understanding how this magically falls back to using the standard golang logging here?

Reply:
Oh nevermind, I looked at the wrapper functions and I see what they're doing.